PyInstaller/utils/hooks/__init__.py

```diff
@@ -1,35 +1,23 @@
-#-----------------------------------------------------------------------------
+# ----------------------------------------------------------------------------
```

Comment:
FWIW, my style checker always highlights these headers, saying [there should be a space](https://www.python.org/dev/peps/pep-0008/#inline-comments) after the `#` that begins a line-comment. I just ignore it and keep the header as-is.